packages/realm/src/Object.ts

```diff
@@ -502,15 +503,19 @@ export class RealmObject<T = DefaultObject, RequiredProperties extends keyof Omi
    *      console.log(` ${prop}`);
    *   });
    * })
+   * @example
+   * wine.addListener((obj, changes) => {
+   *  console.log("The wine got deleted or its brand might have changed");
+   * }, ["brand"])
```

Comment:
Regarding the `@example`'s log message, I believe it's technically also for when the listener is added. (Perhaps not necessary for the `@example` though.)

Reply:
True - but for the sake of simplicity, I suggest leaving it as-is.